ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -234,46 +265,116 @@ then
 fi
 
 if [ -e /root/bm.json ] ; then
```

Comment:
Could be possible at least to move this entire block under a `manage_baremetal_instances` (or any other better name) function? My main concern is that the inner dev-scripts may risk to grow in a way that will be difficult to maintain (probably in future most of the code could be moved directly inside dev-scripts, but at least meanwhile it could be slightly more manageable.

(As a general note, non related to this patch, I think we should try to simplify as much as possible this step, leaving here just the part that strictly needs to interact with the CI)

Reply:
I'm not sure it will help readability as it will all be stuck inside the heredoc but I can move it

Reply:
done